msteams-platform/m365-apps/combine-office-add-in-and-teams-app.md

```diff
@@ -0,0 +1,619 @@
+---
+title: Add an Outlook Add-in to a Teams app
```

Comment:
```suggestion
title: Add Outlook add-in to Teams
```

Reply:
> 

This is not accurate. They are adding an addin to a Teams app, not to Teams. Also, the "a" and "an" are needed to make this idiomatic English.